test/test_collection.py

```diff
@@ -0,0 +1,7 @@
+from pymongo import MongoClient
+from pymongoexplain.explainable_collection import ExplainCollection
+client = MongoClient(serverSelectionTimeoutMS=1000)
+collection = client.db.products
+explain = ExplainCollection(collection)
+print(explain.update_one({"quantity": 1057, "category": "apparel"}, {"$set": {
+    "reorder": True}}))
```

Comment:
Can you turn this into a real test using unittest? https://docs.python.org/3/library/unittest.html#basic-example